vsock-proxy/enclave/src/app_configuration.rs

```diff
@@ -21,8 +23,6 @@ use shared::models::CCMBackendUrl;
 
 use crate::certificate::CertificateResult;
 use crate::enclave::write_to_file;
-use em_app::compute_app_config_hash;
-use em_app::utils::models;
```

Comment:
There are a lot of changes due to `cargo fmt`. Can you split it up in two PRs for clarity?

Reply:
Aditi wanted me to run the fmt https://github.com/fortanix/salmiac/pull/11#discussion_r1796899509, your request kinda contradicts her request.

Reply:
I'm okay to move cargo fmt changes to another PR or if it is a separate commit in this PR.

Reply:
It is a separate commit